UNRELEASED.md

```diff
@@ -10,3 +10,6 @@
          * Some bug fix, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+### Breaking changes
+
+* Rework module lookup (drops support for `TLA_PATH`), see #1491
```

Comment:
Perhaps the support for automatically finding tla modules in the file's directory should also be noted as a new feature?

Reply:
Good point, updated in 180492be